src/ontology/cl-edit.owl

```diff
@@ -4850,10 +4850,10 @@ SubClassOf(obo:CL_0000164 obo:CL_0000163)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D055099"^^xsd:string) obo:IAO_0000115 obo:CL_0000165 "A neuron that is capable of some hormone secretion in response to neuronal signals."^^xsd:string)
 AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_0000165 "CL:0000381")
-AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0000165 "neurosecretory cell"^^xsd:string)
-AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0000165 "neurosecretory neuron"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000165 "BTO:0002691"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000165 "FMA:83810"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/neurosecretion"^^xsd:string) Annotation(rdfs:comment "The modern definition of neurosecretion has evolved to include the release of any neuronal secretory product from a neuron.") oboInOwl:hasRelatedSynonym obo:CL_0000165 "neurosecretory cell"^^xsd:string)
```

Comment:
Recommend setting the dbxref to the exact reference (it looks like it is ISBN:9780323711548). I think the current URL goes to a collection of several sources and those results can change over time.

Reply:
> those results can change over time.
Good point! 

I chose that reference because I'm referencing the synonym, not the comment. However, it is better to provide the source of the comment

Reply:
I think the reason QC checks are failing is because the comment should not be an annotation of an annotation. Consider consolidating this comment with your other comment or try adding a second comment that is not under the synonym annotation.

Reply:
Thank you, I thought there was a problem with the reference but I couldn't find it.